npc/re/guild/mission_main.txt

```diff
@@ -0,0 +1,434 @@
+			next;
+			mes .@npcname$;
+			mes "Guild dungeon daily quests consist of combat quests that you can complete in guild dungeons.";
+			mes "It's simple.";
+			close;
+		case 4: // I don't need it.
+			mes .@npcname$;
+			mes "You're really something else~!";
+			close;
+	}
+	end;
+
+OnInit:
+	setarray .castle$[0],
+		"prtg_cas01","prtg_cas02","prtg_cas03","prtg_cas04","prtg_cas05",
+		"payg_cas01","payg_cas02","payg_cas03","payg_cas04","payg_cas05",
+		"gefg_cas01","gefg_cas02","gefg_cas03","gefg_cas04","gefg_cas05",
+		"aldeg_cas01","aldeg_cas02","aldeg_cas03","aldeg_cas04","aldeg_cas05",
+		"arug_cas01","arug_cas02","arug_cas03","arug_cas04","arug_cas05",
+		"schg_cas01","schg_cas02","schg_cas03","schg_cas04","schg_cas05",
+		"te_aldecas1","te_aldecas2","te_aldecas3","te_aldecas4","te_aldecas5",
+		"te_prtcas01","te_prtcas02","te_prtcas03","te_prtcas04","te_prtcas05";
+	setarray .npcname$[0],
+		"[Altir]","[Alshine]","[Denev]","[Tarazed]","[Mirah]","[Almark]","[Becrux]","[Acrux]";
+	setarray .region$[0],
+		"Valkyrie Realm","Greenwood Lake","Britoria","Luina","Valfreyja","Nidhoggur","Kafragarten","Gloria";
+	setarray .weekday[0], // Day of the week. 0 = Sunday, 6 = Saturday.
+		0,0,0,0,0,0,6,6;
+	setarray .start_time[0],
+		2200,2200,2200,2200,2200,2200,2200,2200;
+	setarray .end_time[0],
+		2230,2230,2230,2230,2230,2230,2230,2230;
+	setarray .conquer_delay_questid[0], // Quest ID for cooldowns
+		7517,7520,7519,7518,7522,7521,7524,7523;
+	setarray .level_req[0], // Level requirement for quests. There are 2-3 quests per region
+		100,120,120,	//prt
+		100,120,120,	//pay
+		100,120,120,	//gef
+		100,120,120,	//alde
+		90,90,0,		//aru
+		90,90,0,		//sch
+		70,70,0,		//tealde
+		70,70,0;		//teprt
+	setarray .max_level[0], // Exp increase cap for quests. 0 for unlimited/not exist
+		130,140,0,	//prt
+		130,140,0,	//pay
+		130,140,0,	//gef
+		130,140,0,	//alde
+		120,120,0,	//aru
+		120,120,0,	//sch
+		99,99,0,	//tealde
+		99,99,0;	//teprt
+	setarray .daily_hunting_questid[0], //Quest ID for daily quests.
+		7525,7526,7527,	//prt
+		7534,7535,7536,	//pay
+		7531,7532,7533,	//gef
+		7528,7529,7530,	//alde
+		7539,7540,0,	//aru
+		7537,7538,0,	//sch
+		7561,7562,0,	//tealde
+		7541,7542,0;	//teprt
+	setarray .daily_delay_questid[0],	//Quest ID for mob hunting check.
+		7543,7544,7545,	//prt
+		7552,7553,7554,	//pay
+		7549,7550,7551,	//gef
+		7546,7547,7548,	//alde
+		7557,7558,0,	//aru
+		7555,7556,0,	//sch
+		7563,7564,0,	//tealde
+		7559,7560,0;	//teprt
+	setarray .p$[0],"first","second","third";
+	end;
+
+	function brief {
+		switch(getarg(0)){
+			case 7527:
+			case 7530:
+			case 7533:
+			case 7536:
+				mes "It is to deal with the powerful creatures that appear in the Corridor of the Abyss.";
+				break;
+			default:
+				mes "It is to eliminate the monsters that appear in the guild dungeon.";
+				break;
+		}
+		return;
+	}
+
+	function mob_brief {
```

Comment:
General about that function:
You should make sure to use plural forms everywhere.